src/main/java/org/jabref/gui/StateManager.java

```diff
@@ -89,4 +92,8 @@ public void clearSearchQuery() {
     public void setSearchQuery(SearchQuery searchQuery) {
         activeSearchQuery.setValue(Optional.of(searchQuery));
     }
+
+    public IntegerProperty searchResultSizeProperty() {
```

Comment:
What about making `searchResultSize` a `ObservableMap<BibDatabaseContext, IntegerProperty>` (similar to `selectedGroups`?

Reply:
Thanks for the hint, works now!